src/Runner.Common/HostContext.cs

```diff
@@ -1,19 +1,18 @@
-﻿using GitHub.Runner.Common.Util;
-using System;
+﻿using System;
```

Comment:
You need to delete where the actions are cached for GHES.... let me find...

Reply:
Check w/ Ting. Make sure this wont interfere with alpha (e.g. already branched)

Reply:
```
            // Clear the cache (for self-hosted runners)
            // Note, temporarily avoid this step for the on-premises product, to avoid rate limiting.
            var configurationStore = HostContext.GetService<IConfigurationStore>();
            var isHostedServer = configurationStore.GetSettings().IsHostedServer;
            if (isHostedServer)
            {
                IOUtil.DeleteDirectory(HostContext.GetDirectory(WellKnownDirectory.Actions), executionContext.CancellationToken);
            }
```